src/poetry/repositories/installed_repository.py

```diff
@@ -58,7 +58,7 @@ def get_package_paths(cls, env: Env, name: str) -> set[Path]:
             if not pth_file.exists():
                 continue
 
-            with pth_file.open() as f:
+            with pth_file.open(encoding="utf-8") as f:
```

Comment:
That might be wrong: #8041

Reply:
Thanks for the feedback, that is now fixed!